runhouse/rns/login.py

```diff
@@ -146,6 +146,15 @@ def login(
         # via configs.token = token
         configs.set("token", token)
 
+    autostop_mins = configs.defaults_cache.get("default_autostop")
+    if autostop_mins is None:
+        new_autostop_mins = typer.prompt(
+            "Set the number of minutes to auto-terminate clusters after a period of inactivity, "
+            "or press `Enter` to set to 60 minutes. Set to -1 to disable autostop entirely.",
```

Comment:
just a suggestion, I feel like could be worded more elegantly but it's kinda tricky. also not sure our stance on static clusters rn either and whether this could be confusing if that's still a use case? 

```suggestion
            "Set the default number of minutes of inactivity after which to auto-terminate clusters. "
            "Press `Enter` to set to 60 minutes, or `-1` to disable autostop entirely.",
```

Reply:
yea couldn't really settle on the best way of wording wording it, and you're right it's really only relevant for on-demand clusters, so thinking we just need to make that more clear in the message